### ISSUE TITLE
[FSR] Content and Design for Additional Info Sections

### ISSUE THREAD

**Issue by mgleo** [department-of-veterans-affairs/va.gov-team#57389]
## Issue Description
Additional info sections are needed throughout the FSR.

-Need to discuss adding in components with Jill Anderson.
-If time allows, our team will take a first pass at creating content for approval by Content team, but not in S98!

Note that the work in this ticket is for the CONTENT Team; not a high priority to complete in S98.

[Based on findings from eFSR study](https://github.com/department-of-veterans-affairs/va.gov-team/blob/5b9b83112fd9f7a7261cc5afe9bd6fa425025438/products/combined_fsr/research/enhanced_fsr/research-findings.md#recommendations).

---
## Tasks
- [x] Draft additional info sections and get internal approval
- [x] Ask for feedback from the content team and incorporate any updates
- [x] Add approved versions to Sketch/UXPin

**Design/Content needed:**

- [x] **[Veteran Employment History > Current job income](https://sketch.com/s/50583485-ba58-4ef8-ae04-2c274a792547/a/God810Y/play)** Add an [Additional Info component](https://design.va.gov/components/additional-info#examples) that explains how to calculate monthly income
- [x] **[Veteran employment history > payroll deductions](https://sketch.com/s/50583485-ba58-4ef8-ae04-2c274a792547/a/mPKORgz/play)**  Add an [Additional Info component](https://design.va.gov/components/additional-info#examples) that explains how to check for payroll deductions and calculate them from paystub
- [x] **[Veteran employment history > payroll deduction amounts](https://sketch.com/s/50583485-ba58-4ef8-ae04-2c274a792547/a/DEw0kDq/play)**  Add an [Additional Info component](https://design.va.gov/components/additional-info#examples) that explains how to check for payroll deductions and calculate them from paystub

## Acceptance Criteria
- [x] Additional info sections drafted and approved internally
- [x] Feedback from the content team has been incorporated
- [ ] Approved versions added to Sketch/UXPin

**Comment by HeatherRienks** [comment]
Discussed with Jill in 5/3/23 Streamlined Waiver sync- we will plan to wrap this work into the Streamlined Waiver efforts.

**Comment by josephrlee** [comment]
Here are the approved drafted additional info pieces. Will be asking the content team for feedback on this.
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc2srIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--9c4bdd3efe95f4dc04acd7b878a2476d4a1afd38/Screenshot%202023-08-25%20at%201.58.51%20PM.png)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc3crIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--c99049aa228116ec0a3a6a050aaf6d20af6e9171/Screenshot%202023-08-25%20at%201.59.25%20PM.png)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBczArIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b4e0b830a1c319173a424a69d57afb342e1e62e6/Screenshot%202023-08-25%20at%201.59.48%20PM.png)

**Comment by HeatherRienks** [comment]
FYI- Joe is attending office hours with Platform this week to get feedback. Denise has approved.

**Comment by josephrlee** [comment]
We also got good feedback on the table inside of an additional-info component. We came up with the solution of [adding an aria-label](https://dsva.slack.com/archives/C01DBGX4P45/p1692905211095819) for the table describing it in-depth. I've [drafted content](https://github.com/department-of-veterans-affairs/va.gov-team/issues/59918#issuecomment-1698099439) and have submitted it for platform feedback. They will help us with editing and the technical implementation.
